app/src/main/cpp/src/components/transport_manager/include/transport_manager/bluetooth_le/bluetooth_le_device_scanner.h

```diff
@@ -89,11 +89,8 @@ class BluetoothLeDeviceScanner : public DeviceScanner {
   void ProcessMessage(const std::vector<uint8_t>& data);
 
   TransportAdapterController* controller_;
-  
-  std::vector<bdaddr_t> paired_devices_;
 
-  DeviceVector paired_devices_with_sdl_;
-  DeviceVector found_devices_with_sdl_;
+  DeviceVector paired_devices_;
```

Comment:
@VadymLuchko ble devices does not require pairing for making a connection, so its better to use name `found_devices_with_sdl_`

Reply:
fixed in 22caff304f02304d55c097802b850d8bca25a370